index.d.ts

```diff
@@ -49,6 +49,15 @@ export interface ActionCmd<A extends Action> {
   simulate(): A;
 }
 
+export interface DelayCmd<A extends Action> {
```

Comment:
i wouldn't  worry about flow. i think it makes sense to drop support for it

Reply:
Okay great. I will remove flow changes.

Reply:
sounds good!

Reply:
Flow changes removed.